Sources/peertalk/PTChannel.m

```diff
@@ -1,5 +1,5 @@
-#import "PTChannel.h"
-#import "PTPrivate.h"
+#import "include/PTChannel.h"
```

Comment:
Is this a necessary change to make things work with Swift PM? If no then, can we keep the scope of this change to adding Swift PM and not moving these headers out too?